shared/fixes/bash/bootloader_nousb_argument.sh

```diff
@@ -0,0 +1,6 @@
+USB_KEYBOARD=$(grep 'Product=' /proc/bus/usb/devices 2>/dev/null| egrep -ic '(ps2 to usb adapter|keyboard|kvm|sc reader)')
+if [ "${USB_KEYBOARD}" = "0" ]; then
+	/sbin/grubby --update-kernel=ALL --args="nousb"
```

Comment:
Is `--args` persistent or will it get ignored when new kernel is shipped? I think `--args` is a one-off option and will be ignored but I am not sure.